landkreise/get-nienburg-kreis.py

```diff
@@ -0,0 +1,29 @@
+from bs4 import BeautifulSoup
+
+import requests
+import datetime
+import scrape
+import locale
+import json
+
+locale.setlocale(locale.LC_TIME, "de_DE.utf-8")
+
+from database_interface import *
+
+main_url = "https://webgis.lk-ni.de/arcgis/rest/services/opengov/Corona_StatusQuo/MapServer/0/query?f=json&where=1%3D1&returnGeometry=false&spatialRel=esriSpatialRelIntersects&outFields=*&orderByFields=Tagesdatum%20desc"
+
+req = scrape.request_url(main_url)
+bs = BeautifulSoup(req.text, "html.parser")
+text = bs.get_text()
+
+#get json object
+json_data = json.loads(text)
+
+cases = json_data["features"][0]["attributes"]["Faelle_bestaetigt"]
+unix_timestamp = int(json_data["features"][0]["attributes"]["Tagesdatum"])
+unix_timestamp /= 1000
+
+#convert unix to date
+status = datetime.datetime.fromtimestamp(unix_timestamp).strftime("%Y-%m-%d")
+
+add_to_database("03256022", status, cases, "Landkreis Nienburg/Weser")
```

Comment:
This is about the landkreis, not the city; right? Then the AGS is actually 03256 . The AGS you wrote is from the city Nienburg/Weser.

Reply:
Thanks. Source?

Reply:
For example here on wikipedia: https://de.wikipedia.org/wiki/Landkreis_Nienburg/Weser

or in the csv/json linked in the readme (note the different strings in the beginning of each line there).

Out of curiosity: How did you find the project?

Reply:
Tracking the requested websites/data api's when loading the website gave me a clue.